src/app/InteractionModelEngine.cpp

```diff
@@ -314,7 +314,8 @@ CHIP_ERROR InteractionModelEngine::OnMessageReceived(Messaging::ExchangeContext
 
 void InteractionModelEngine::OnResponseTimeout(Messaging::ExchangeContext * ec)
 {
-    ChipLogProgress(DataManagement, "Time out! failed to receive echo response from Exchange: %d", ec->GetExchangeId());
+    ChipLogProgress(DataManagement, "InteractionModelEngine:Time out! failed to receive echo response from Exchange: %d",
```

Comment:
Shouldn't we change the log category, rather than editing the string to add more context?

Reply:
Acked.